antsibull/data/collection-enum.py

```diff
@@ -124,6 +128,39 @@ def load_all_plugins(plugin_type, basedir, coll_filter):
     return result
 
 
+def load_collection_meta_manifest(b_manifest_path, data):
+    with open(b_manifest_path, 'rb') as f:
+        meta = json.load(f)
+    data['namespace'] = meta['collection_info']['namespace']
+    data['name'] = meta['collection_info']['name']
+    data['version'] = meta['collection_info']['version']
```

Comment:
The approach of modifying incoming arguments by reference causes C flashbacks. It's hard to read and maintain. Python doesn't have the same constraints as C does so it's strongly discouraged to use this approach.
I recommend using `return` because on the caller's side `make_a_function_call()` does not look like this thing is going to influence the state outside of its local scope which would be surprising.

Reply:
Fixed.